harness/determined/deploy/aws/templates/efs.yaml

```diff
@@ -695,6 +709,11 @@ Resources:
             EOF
             fi
 
+            efs_id=${EFSID}
```

Comment:
could we, instead of adding to bash logic, do it in plain cloudformation substitution instead?
i.e. 
1. Use one name (.e.g) ${EFS_VAL} in the template. 
2. add circa line 866 here:

```
...
- AgentAmi: !FindInMap [RegionMap, !Ref "AWS::Region", Agent]
- EFS_VAL !If [CreateEFS, !Ref EFS, !Ref EFSID]
...
```